website/content/commands/debug.mdx

```diff
@@ -71,14 +71,14 @@ all targets for 2 minutes.
+| --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `agent`   | Version and configuration information about the agent.                                                                                                                                                                                                                                                                                                                                                                    |
+| `host`    | Information about resources on the host running the target agent such as CPU, memory, and disk.                                                                                                                                                                                                                                                                                                                           |
+| `cluster` | A list of all the WAN and LAN members in the cluster.                                                                                                                                                                                                                                                                                                                                                                     |
+| `metrics` | Metrics from the in-memory metrics endpoint in the target, captured at the interval.                                                                                                                                                                                                                                                                                                                                      |
+| `logs`    | `DEBUG` level logs for the target agent, captured for the duration.                                                                                                                                                                                                                                                                                                                                                       |
+| `pprof`   | Golang heap, CPU, goroutine, and trace profiling. CPU and traces are captured for `duration` in a single file while heap and goroutine are snapshots for each `interval` captured in multiple files for `duration`. This information is not retrieved unless [`enable_debug`](/docs/agent/options#enable_debug) is set to `true` on the target agent or ACL are enable and an ACL token with `operator:read` is provided. |
```

Comment:
```suggestion
| `pprof`   | Golang heap, CPU, goroutine, and trace profiling. CPU and traces are captured for `duration` in a single file while heap and goroutine are separate snapshots for each `interval`. This information is not retrieved unless [`enable_debug`](/docs/agent/options#enable_debug) is set to `true` on the target agent or ACLs are enable and an ACL token with `operator:read` is provided. |
```

Ha ha I know I said this was a good place for detail 😄 I think the "multiple files for duration" part might actually make it harder to understand without a little more clarification which might be overkill here. This is slightly _less_ specific but I think clear enough. What do you think?